Assets/MRTK/SDK/Features/Input/Handlers/Constraints/TransformConstraint.cs

```diff
@@ -54,30 +41,18 @@ public ManipulationProximityFlags ProximityType
             set => proximityType = value;
         }
 
-        protected MixedRealityPose worldPoseOnManipulationStart;
+        protected MixedRealityTransform worldPoseOnManipulationStart;
 
         public abstract TransformFlags ConstraintType { get; }
 
         #endregion Properties
 
-        #region MonoBehaviour Methods
-
-        public virtual void Start()
-        {
-            if (TargetTransform == null)
-            {
-                TargetTransform = transform;
-            }
-        }
-
-        #endregion MonoBehaviour Methods
-
         #region Public Methods
 
         /// <summary>
         /// Intended to be called on manipulation started
         /// </summary>
-        public virtual void Initialize(MixedRealityPose worldPose)
```

Comment:
Since this might have been overridden in custom scripts, can it be left around with the obsolete tag (and recommendation to update the parameter type), then call right into the new `MixedRealityTransform`-based method?